### ISSUE TITLE
S3 content length limit (8388608 bytes) Error with 0.20 version

### ISSUE THREAD

**Issue by pramodlahoti26** [akka/alpakka#1176]
we are getting Error as exceeded content length limit (8388608 bytes)! You can configure this by setting `akka.http.[server|client].parsing.max-content-length` 
or calling `HttpEntity.withSizeLimit` before materializing the dataBytes stream 

As per latest commits it shows this issue has been addressed with 0.20 and we are using latest version but still getting same issue ,

Is there a way we can explicitly set this property externally or any other workaround to set this property?

Code we are using to Download  - 
```s3Client.download(bucketName.get, content.key)._1.
via(Framing.delimiter(ByteString("\n"), Constant.BYTE_SIZE, byeFormatterFlag).map(_.utf8String))
```

**Comment by Learner0426** [comment]
StackOverflow Details about Issue - https://stackoverflow.com/questions/52111284/alpakka-akka-http-failure-while-downloading-file-with-latest-s3-connector

**Comment by gabfssilva** [comment]
You can just configure it in your application.conf inside src/main/resources:

```
akka {
  http {
    client {
      parsing {
        max-content-length = //max length limit in bytes you want to set
      }
    }
  }
}
```

A good observation is that this property does not mean the max size of data that Akka HTTP keeps in memory.

**Comment by 2m** [comment]
Did you try increasing the `akka.http.client.parsing.max-content-length` parameter?

**Comment by ennru** [comment]
Fixed with #1036
